### ISSUE TITLE
cp1616: set_alarm.h missing?

### ISSUE THREAD

**Issue by gavanderhoorn** [ros-industrial/siemens_experimental#9]
Trying to compile this in a catkin workspace gets me the following error:

```
/path/to/catkin_ws/src/siemens_experimental/siemens_cp1616/src/siemens_cp1616_io_device_wrapper.cpp:24:38: fatal error: siemens_cp1616/set_alarm.h: No such file or directory
```

Is this file still needed? Could it be added to the repository?

**Comment by gavanderhoorn** [comment]
Ah, that seems to be a header for a service. The `CMakeLists.txt` seems to be missing the required statements for generation of the header files though.

**Comment by gavanderhoorn** [comment]
This is fixed by durovsky/siemens_experimental#4.
